Source/engine.cpp

```diff
@@ -26,6 +26,12 @@ const int RndInc = 1;
  */
 const int RndMult = 0x015A4E35;
 
+/**
+ * @brief Find the start of a CEL frame
+ * @param pCelBuff Cel data
+ * @param nCel Frame of cel

```

Comment:
```suggestion
 * @param nCel CEL frame number
```

The same goes for other such docs in this file.